### DISCUSSION TITLE
a little bug in command mode

### DISCUSSION THREAD

**Discussion by oeyoews** [nvim-neorg/neorg#264] [General]
![2021-12-20 21-48-56 的屏幕截图](https://user-images.githubusercontent.com/72405338/146778747-8bd7f970-74b7-4c19-9fb9-c1818dc77588.png)
- If I type a space at the beginning, it will continue to complete it in a loop afterwards

**Comment by vhyrro** [comment]
Hey! Sorry for not responding for a while, I missed this discussion. It should be fixed now, let me know if the issue is still present :)
